File: QID-2455-MVAbankrupt/MVAbankrupt.r

# clear all variables
rm(list = ls(all = TRUE))
graphics.off()

# read data and set variables
data  = read.table("bankrupt.txt")
length(data)
y     = data[, 6]
x1    = data[, 1]
x2    = data[, 2]
x3    = data[, 3]
x4    = data[, 4]
x5    = data[, 5]

# compute logit model
fit   = glm(y ~ x3 + x4 + x5, family = binomial(link = "logit"))
summary(fit)

File: QID-2455-MVAbankrupt/MVAbankrupt.py
#works on pandas 1.5.3 and statsmodels 0.14.0
import pandas as pd
import statsmodels.api as sm

data = pd.read_table("bankrupt.txt", sep="\s+", header=None, names = [1,2,3,4,5,6]) 
y = data.loc[:, 6]
X = sm.add_constant(data.loc[:,[3,4,5]])
fit = sm.Logit(y, X).fit()

print(fit.summary())